### PR TITLE
fix test-and-docs typo "do" to "due"

### PR DESCRIPTION
**Pull request by ghost**
Another small fix :smiley: hope you don't mind these super small changes!

I just see it as I'm going through it the first time & fix it.

### CODE REVIEW

**Comment by khoomeister** [comment]
Whoops - I made the change!

File: docs/sources/project/test-and-docs.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ To run the same test inside your Docker development container, you do this:
 
     root@5f8630b873fe:/go/src/github.com/docker/docker# TESTFLAGS='-run ^TestBuild$' hack/make.sh
 
-## If test under Boot2Docker fail do to space errors
+## If test under Boot2Docker fail due to disk space errors
```
Comment (moxiegirl) [inline docs/sources/project/test-and-docs.md]: Yowza and I spotted another problem test > tests

"If tests under Boot2Docker fail due to disk space errors"

Can you add that to your fix? I'll LGTM it.  Thanks for the catch --- small or large problems we want 'em all!

**Comment by khoomeister** [comment]
Done and dusted :smiley:

**Comment by moxiegirl** [comment]
You rock! Thanks so much. LGTM

**Comment by moxiegirl** [comment]
@khoomeister Your PR is merged...you can see your change at: http://docs.master.dockerproject.com/

**Pull request merged**